nx-X11/programs/Xserver/hw/nxagent/NXdixfonts.c

```diff
@@ -999,1393 +622,86 @@ doListFontsAndAliases(ClientPtr client, LFclosurePtr c)
     return TRUE;
 }
 
+
 int
-ListFonts(ClientPtr client, unsigned char *pattern, unsigned length, 
```

Comment:
> In ListFonts? No, there's the nxagentMaxFontNames change (I haven't actually checked what it's 
> set to, but I assume it might be a higher value than the hardcoded "default" of 100.)

Ack. Saw it now. Again a one line change. nxagentMaxFontNames is set to 10000 btw. I pushed a fix-up for this, as well.